app/src/main/java/com/example/cristiana/weatherapp/TenDaysActivity.java

```diff
@@ -0,0 +1,212 @@
+                } else {
+                    Toast.makeText(TenDaysActivity.this, "onResponse was unsuccessful", Toast.LENGTH_SHORT).show();
+                }
+            }
+
+            @Override
+            public void onFailure(Call<TenDaysWeather> call, Throwable t) {
+                t.printStackTrace();
+                Toast.makeText(TenDaysActivity.this, "onFailure", Toast.LENGTH_SHORT).show();
+            }
+        });
+    }
+
+    static class Adapter extends RecyclerView.Adapter<RecyclerView.ViewHolder> {
+        List<Forecast> mForecast = new ArrayList<>();
+        Callback mCallback;
+
+        public Adapter(Callback callback) {
+            mCallback = callback;
+        }
+
+        public interface Callback {
+            void show (Forecast forecast);
+        }
+
+        public void setData(TenDaysWeather tenDaysWeather) {
+            List<Forecast> forecasts = tenDaysWeather.getForecast();
+            for (Forecast forecast : forecasts) {
+                mForecast.add(forecast);
+            }
+        }
+
+        @Override
+        public RecyclerView.ViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+            View view = LayoutInflater.from(parent.getContext()).inflate(R.layout.view_day, parent, false);
+            return new ViewHolder(view);
+        }
+
+        @Override
+        public void onBindViewHolder(RecyclerView.ViewHolder holder, final int position) {
+            ((ViewHolder) holder).bind(mForecast.get(position), new View.OnClickListener() {
+
+                @Override
+                public void onClick(View v) {
+                    mCallback.show(mForecast.get(position));
+                }
+            });
+        }
+
+        @Override
+        public int getItemCount() {
+            return mForecast.size();
+        }
+
+        static class ViewHolder extends RecyclerView.ViewHolder {
+            private final TextView mCrtDate;
+            private final TextView mCrtTemperature;
+
+            public ViewHolder(View itemView) {
+                super(itemView);
+
+                mCrtDate = (TextView) itemView.findViewById(R.id.crtDay);
+                mCrtTemperature = (TextView) itemView.findViewById(R.id.crtTemperature);
+            }
+
+            public void bind(Forecast forecast, View.OnClickListener onClickListener) {
+                String[] months = {"January", "February", "March", "April", "May", "June", "July",
+                                    "August", "September", "October", "November", "December"};
+                DateFormat mDateFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss", Locale.ENGLISH);
+                Calendar calendar = Calendar.getInstance();
+
+                try {
+                    Date mDate = mDateFormat.parse(forecast.getDtTxt());
+                    calendar.setTime(mDate);
+                    mCrtDate.setText(months[calendar.get(Calendar.MONTH)] + " " +  calendar.get(Calendar.DAY_OF_MONTH)
```

Comment:
Using a SimpleDateFormat for the output will also help you in cases where this prints 12h:1m instead of 12h:01m